pkg/apis/greenhouse/v1alpha1/organization_types.go

```diff
@@ -85,11 +85,11 @@ type SCIMConfig struct {
 	// +kubebuilder:default="basic"
 	AuthType scim.AuthType `json:"authType,omitempty"`
 	// User to be used for basic authentication.
-	BasicAuthUser ValueFromSource `json:"basicAuthUser"`
+	BasicAuthUser *ValueFromSource `json:"basicAuthUser,omitempty"`
 	// Password to be used for basic authentication.
-	BasicAuthPw ValueFromSource `json:"basicAuthPw"`
+	BasicAuthPw *ValueFromSource `json:"basicAuthPw,omitempty"`
 	// BearerToken to be used for bearer token authorization
-	BearerToken ValueFromSource `json:"bearerToken"`
+	BearerToken *ValueFromSource `json:"bearerToken,omitempty"`
 	// BearerPrefix to be used to defined bearer token prefix
 	// +kubebuilder:default:=Bearer
```

Comment:
The `+kubebuilder:default` annotations for BearerPrefix and BearerHeader should be removed.
This is optional and only needs to be used if the defaults used by the controller are not applicable:
https://github.com/cloudoperators/greenhouse/pull/869/files#diff-9ff0404ba7b378df0af6af68c45c832020fc20d6781b71f35179aa58328b20b5R71-R79